<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="connect_Button">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>发包</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="recvButton">
      <property name="text">
       <string>收包</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>186</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="sendheartButton">
      <property name="text">
       <string>发心跳包</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="floodButton">
      <property name="text">
       <string>flood攻击</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
